src/unity/python/turicreate/test/test_sarray.py

```diff
@@ -3168,24 +3168,20 @@ def test_filter_by(self):
         
         #integer example
         x = SArray([1,2,3,4,5,6,7])
-        self.assertTrue(np.array_equal(x.filter_by([11,7,2,8,4]), SArray([2,4,7])))
-        self.assertTrue(np.array_equal(x.filter_by([11,7,2,8,4,3], exclude=True), SArray([1,5,6])))
+        self.assertTrue(sorted(x.filter_by([11,7,2,8,4])) == [2,4,7])
```

Comment:
Now that this uses the == operator, we'll get better test output on failure using `self.assertEqual` instead.

Reply:
Duh, good catch

Thanks!